commercial/app/model/commercial/CapiImages.scala

```diff
@@ -0,0 +1,58 @@
+package model.commercial
+
+import views.support.ImgSrc
+import cards.Standard
+import layout.{FaciaWidths, ItemClasses}
+import model.ImageMedia
+import play.api.libs.json.{Json, Writes}
+
+object CapiImages {
+
+  // Puts together image source info using data from cAPI.
+  def buildImageData(imageData: Option[ImageMedia]): ImageInfo = {
+
+    val fallbackImageUrl = imageData flatMap ImgSrc.getFallbackUrl
+    val cardType = Standard
+
+    val breakpointWidths = FaciaWidths.mediaFromItemClasses(ItemClasses(
+      mobile = Standard,
```

Comment:
Turns out something I thought we'd be able to do client-side was actually a bit more complicated.

The image data depends on the number of cards in a couple different ways (didn't realise this before as it's buried quite deeply). Should only affect the occasions when there are fewer than 4 cards available, but need to handle it regardless: 47ac760.